app/org/sagebionetworks/bridge/dynamodb/DynamoHealthDataRecord.java

```diff
@@ -295,6 +298,19 @@ public void setUserDataGroups(Set<String> userDataGroups) {
         // DDB doesn't support empty sets, use null reference for empty set. This is also enforced by the builder.
         this.userDataGroups = (userDataGroups != null && !userDataGroups.isEmpty()) ? userDataGroups : null;
     }
+    
+    /** {@inheritDoc} */
+    @DynamoDBTypeConvertedJson
```

Comment:
I think there's still a misunderstanding here. Why do we need JSON at all? DynamoDB handles String maps just fine.

Reply:
Not according to the integration tests. Without this annotation the map is always empty.

Reply:
It seems to work just fine for DynamoStudy.getInstallLinks

Reply:
Verified (some part of BridgePF, BridgeJavaSDK and/or BridgeIntegrationTests must not have been in the right branch or updated when I tested this the first time... it persists fine).